packages/block-library/src/table/edit.js

```diff
@@ -456,6 +456,9 @@ function TableEdit( {
 					<BlockControls group="block">
 						<AlignmentControl
 							label={ __( 'Change column alignment' ) }
```

Comment:
I'd change this label to use a more consistent wording with the other labels. Something along the lines of 'Align columns content' would be clear enough to amke the description not necessary.